### ISSUE TITLE
Replace custom progress bar with accessible Slider

### ISSUE THREAD

**Issue by Bender21m** [satsdisco/JellyAmp#25]
NowPlayingView progress bar is a GeometryReader+DragGesture hack. Should be a styled Slider for better VoiceOver support and standard iOS scrubbing feel.

**Comment by Bender21m** [comment]
Done in a77cb38
